.eslintrc.yml

```diff
@@ -43,3 +45,7 @@ rules:
   flowtype/type-id-match:
   - 1
   - "^([A-Z][a-z0-9]+)+Type$"
+
+  react/jsx-filename-extension:
+    - 1
+    - extensions: [".js", ".jsx"]
```

Comment:
@justin808 It should be:

```yml
react/jsx-filename-extension:
  - 1
  - extensions: [".js"]

# or

react/jsx-filename-extension:
  - 1
  - extensions: [".jsx"]

```

In general I'm fine w/ either, but as we already have a big codebase, that's using `.jsx`, I'd stay were we are as there's no reason to spend time on this change.